AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -32,6 +33,7 @@ struct ConfirmCreatingSessionView: View {
                         
             Button(action: {
                 sessionContext.setupAB()
+                didStartRecordingSession = true
```

Comment:
Do we use this anywhere?

Reply:
We don't, nice catch, thanks. I'm removing it.